src/main/java/com/datapipe/jenkins/vault/VaultBuildWrapper.java

```diff
@@ -136,10 +136,18 @@ protected void provideEnvironmentVariablesFromVault(Context context, Run build,
         }
 
         VaultConfig vaultConfig = config.getVaultConfig();
-
         VaultCredential credential = config.getVaultCredential();
         if (credential == null) credential = retrieveVaultCredentials(build);
 
+        String prefixPath = "";
+        if (config.getPrefixPath() == null || config.getPrefixPath().isEmpty()) {
+            logger.printf("No prefixPath provided%n");
+        } else {
+            prefixPath = config.getPrefixPath();
+            vaultConfig.prefixPath(prefixPath);
+            logger.printf("Using prefixPath with Vault: %s%n", config.getPrefixPath());
+        }
```

Comment:
Could be shorted to:

```suggestion
        String prefixPath = StringUtils.isBlank(config.getPrefixPath())
            ? ""
            : Util.ensureEndsWith(envVars.expand(config.getPrefixPath()), "/");
        vaultConfig.prefixPath(prefixPath);
```